### DISCUSSION TITLE
Do you have any schedule to release a new version?

### DISCUSSION THREAD

**Discussion by AlexXiong-dev** [zxing-cpp/zxing-cpp#580] [Q&A] [answered]
The latest version is v2.0.0 which released on Jan 6, so do you have any schedule to release a new version to include latest commits?
Thanks.

**Comment by axxel** [comment]
I don't have a specific schedule but as I hinted on in #577, I'm coming closer to the time where it feels like a good idea to make the next release. Is there anything specific in current HEAD that you are "waiting for"?

**Reply by AlexXiong-dev** [reply]
We are waiting for https://github.com/zxing-cpp/zxing-cpp/pull/576 and https://github.com/zxing-cpp/zxing-cpp/pull/567.
It is not so urgent, we can wait.
Thanks.

**Reply by axxel** [reply, marked as answer]
6 Months later to the day I made the next release...

**Reply by AlexXiong-dev** [reply]
Good news. Thanks very much.
